server/src/main/java/com/cloud/network/lb/LoadBalancingRulesManagerImpl.java

```diff
@@ -2189,13 +2190,16 @@ public Pair<List<? extends UserVm>, List<String>> listLoadBalancerInstances(List
         Boolean applied = cmd.isApplied();
 
         if (applied == null) {
+            s_logger.debug(String.format("The [%s] parameter was not passed. Using the default value [%s].", ApiConstants.APPLIED, Boolean.TRUE));
             applied = Boolean.TRUE;
         }
 
         LoadBalancerVO loadBalancer = _lbDao.findById(loadBalancerId);
         if (loadBalancer == null) {
+            s_logger.warn(String.format("Unable to find the load balancer with ID [%s].", cmd.getId()));
             return null;
         }
+        String loadBalancerAsString = ReflectionToStringBuilderUtils.reflectOnlySelectedFields(loadBalancer, "uuid", "name");
```

Comment:
We could implement `toString` of `LoadBalancerVO`. This way we can call `toString` instead of repeating the same code wherever it is used.